site/content/entry/citra-progress-report-2023-h2/index.md

```diff
@@ -0,0 +1,328 @@
++++
+date = "2024-01-25T20:00:00+01:00"
+title = "Citra Progress Report 2023 H2"
+tags = [ "progress-report" ]
+author = "emufan_4568"
+coauthor = "PabloMK7"
+forum = 845615
```

Comment:
```suggestion
forum = 1013527
```
Opened a post for this article, will be listed once this is merged
https://community.citra-emu.org/t/citra-progress-report-2023-h2/1013527